File: app/src/main/java/bassamalim/hidaya/database/AyatDao.java
package bassamalim.hidaya.database;

import androidx.room.Dao;
import androidx.room.Query;

import java.util.List;

@Dao
public interface AyatDao {

    @Query("SELECT * FROM hafs_ayat")
    List<AyatDB> getAll();

/*
    @Query("SELECT * FROM hafs_ayat WHERE id IN (:userIds)")
    List<JAyah> loadAllByIds(int[] userIds);

    @Query("SELECT * FROM user WHERE first_name LIKE :first AND " +
            "last_name LIKE :last LIMIT 1")
    JAyah findByName(String first, String last);

    @Insert
    void insertAll(JAyah... users);

    @Delete
    void delete(JAyah user);
    */
}

File: app/src/main/java/bassamalim/hidaya/data/database/daos/AyatDao.kt
package bassamalim.hidaya.data.database.daos

import androidx.room.Dao
import androidx.room.Query
import bassamalim.hidaya.data.database.dbs.AyatDB

@Dao
interface AyatDao {
    @Query("SELECT * FROM ayat")
    fun getAll(): List<AyatDB>
}